x-pack/solutions/security/plugins/security_solution/public/asset_inventory/components/filters/filters.tsx

```diff
@@ -0,0 +1,198 @@
+    () =>
+      createKbnUrlStateStorage({
+        history,
+        useHash: false,
+        useHashQuery: false,
+      }),
+    [history]
+  );
+  const filterControlsUrlState = useMemo(
+    () =>
+      urlStorage.get<FilterControlConfig[] | undefined>(URL_PARAM_KEY.assetInventory) ?? undefined,
+    [urlStorage]
+  );
+
+  const setFilterControlsUrlState = useCallback(
+    (newFilterControls: FilterControlConfig[]) => {
+      urlStorage.set(URL_PARAM_KEY.assetInventory, newFilterControls);
+    },
+    [urlStorage]
+  );
+
+  const dataViewSpec = useMemo(
+    () =>
+      indexPattern
+        ? {
+            id: SECURITY_ASSET_INVENTORY_DATA_VIEW.id,
+            name: SECURITY_ASSET_INVENTORY_DATA_VIEW.name,
+            allowNoIndex: true,
+            title: indexPattern.title,
+            timeFieldName: '@timestamp',
+          }
+        : null,
+    [indexPattern]
+  );
+
+  const [loadingPageFilters, setLoadingPageFilters] = useState(true);
+
+  // TODO Resolve when integrating with backend
+  // const { dataView, isLoading: isLoadingDataView } = useAlertsDataView({
+  //   ruleTypeIds,
+  //   dataViewsService: dataViews,
+  //   http,
+  //   toasts,
+  // });
+  const isLoadingDataView = false;
+
+  useEffect(() => {
+    if (!isLoadingDataView) {
+      // If a data view spec is provided, create a new data view
+      if (dataViewSpec?.id) {
+        (async () => {
+          // Creates an adhoc data view starting from the alert data view
+          // and applying the overrides specified in the dataViewSpec
+          const spec = {
+            ...(dataView ?? {}),
+            ...(dataViewSpec ?? {}),
+            // } as DataViewSpec;
+          } as unknown as DataViewSpec;
+          await dataViews.create(spec);
+          setLoadingPageFilters(false);
+        })();
+      } else {
+        setLoadingPageFilters(false);
+      }
+    }
+
+    return () => dataViews.clearInstanceCache();
+  }, [dataView, dataViewSpec, dataViews, isLoadingDataView]);
+
+  const handleFilterChanges = useCallback(
+    (newFilters: Filter[]) => {
+      if (!onFiltersChange) {
+        return;
+      }
+      const updatedFilters = newFilters.map((filter) => {
+        return {
+          ...filter,
+          meta: {
+            ...filter.meta,
+            disabled: false,
+          },
+        };
+      });
+
+      onFiltersChange(updatedFilters);
+    },
+    [onFiltersChange]
+  );
+
+  if (!spaceId || !dataViewSpec) {
+    return null;
+  }
+
+  if (loadingPageFilters) {
+    return (
+      <EuiFlexItem grow={true}>
+        <FilterGroupLoading />
+      </EuiFlexItem>
+    );
+  }
+
+  return (
+    <>
+      <EuiSpacer size="l" />
+      <FilterGroup
+        dataViewId={dataViewSpec?.id || null}
+        onFiltersChange={handleFilterChanges}
+        ruleTypeIds={ASSET_INVENTORY_RULE_TYPE_IDS}
+        Storage={Storage}
+        defaultControls={DEFAULT_ASSET_INVENTORY_FILTERS}
+        chainingSystem="HIERARCHICAL"
```

Comment:
nit: change to a const unless this is a string union type
```suggestion
        chainingSystem={HIERARCHICAL}
```